spec/features/user/user_dashboard_spec.rb

```diff
@@ -87,8 +87,100 @@
 
     expect(current_path).to eq(dashboard_path)
 
-    within '#github' do
-      expect(page).to_not have_content('Github Repositories')
+
+    expect(page).to_not have_content('Github')
+    expect(page).to_not have_content('Repositories')
+    expect(page).to_not have_content('Followers')
+    expect(page).to_not have_content('Following')
+  end
```

Comment:
This is a better way than writing 2 or 3 extra tests and falls more in line with our implementation.